doc/cmake.txt

```diff
@@ -209,6 +210,32 @@ cq			Close the CMake console window.
 
 <C-C>			Stop the current command.
 
+==============================================================================
+Events							*cmake-events*
+
+To customize behaviour after a command has finished, Vim-CMake calls user
```

Comment:
I would slightly edit this into

To customize *the* behaviour after a *`:CMakeBuild`* command has finished, Vim-CMake *defines some* user *events*.

(the words in italics are just to point at the changes)